### PR TITLE
create mouse capture example

### PR DESCRIPTION
**Pull request by harbdog**
# What issue is this addressing?
Example used for #2634

## What _type_ of issue is this addressing?
bug

## What this PR does | solves
Provides an example demo for mouse capture

### CODE REVIEW

File: examples/capture/main.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
+// Copyright 2018 The Ebiten Authors
```
Comment (TotallyGamerJet) [inline examples/capture/main.go:1]: Should be 2023

File: examples/capture/main.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 The Ebiten Authors
```
Comment (hajimehoshi) [inline examples/capture/main.go:1]: Ebitengine

**Comment by hajimehoshi** [comment]
I'm not sure the purpose of this PR. This PR does not fix the bug, right?

**Comment by hajimehoshi** [comment]
Let me close this as #2634 was resolved.

**Comment by harbdog** [comment]
Sorry it was orignally to provide an example for that bug, so I've opened instead as feature enhancement now: #2636

**Pull request closed without merging**